bin/boilerplate_template.js

```diff
@@ -0,0 +1,41 @@
+import { createRequire } from 'module'
+const require = createRequire(import.meta.url)
+const dictionary = require('./request_schema.json')
+
+export default {
```

Comment:
could you also put the full example of the template in a mdx file for those who want to run it in devtools? The mdx file could be generated from this boilerplate_template file too.